FuelGaugeReader/FuelGaugeReader.ino

```diff
@@ -0,0 +1,38 @@
+#define USE_USBCON
+#include <ros.h>
+#include <std_msgs/Int8.h>
+std_msgs::Int8 percent_full;
+ros::Publisher pub_percent_full("percent_full", &percent_full);
+ros::NodeHandle nh;
+
+void setup()
+{
+  nh.initNode();
+  Serial.begin(9600);
+  nh.getHardware()->setBaud(9600);
+  nh.advertise(pub_percent_full);
+}
+
+long publisher_timer;
```

Comment:
This and READ_PIN look to be defined in the global space but are between setup and main, this is confusing. put the declarations at the top. Also can you change the name READ_PIN to something more meaningful :)

Reply:
Fixified